docs/website/pages/docs/developer/_meta.en-US.json

```diff
@@ -1,4 +1,5 @@
 {
   "installation": "Installation",
-  "rooch-cli": "Rooch CLI"
+  "rooch-cli": "Rooch CLI",
+  "rtfsc": "RTFSC"
```

Comment:
RTFSC means？ We need a document group to introduce stdlib and framework. This document should be part of it.